app/policies/proposal_policy.rb

```diff
@@ -13,24 +14,32 @@ def initialize(user, proposal)
     @proposal = proposal
   end
 
-  def author?
+  def is_author?
     @proposal.requester_id == @user.id
   end
 
-  def not_approved?
+  def is_not_approved?
     !@proposal.approved?
   end
 
-  def approve_reject?
+  def is_approver?
+    !@proposal.approvals.find_by(user: @user).nil?
+  end
+
+  def is_pending_approver?
```

Comment:
Okidok. I was trying to match the can_XXX names, but I can definitely strip off the is'es